### PR TITLE
Feature/game detail#134

### PR DESCRIPTION
**Pull request by wsite**

### CODE REVIEW

File: client/static/app/modules/bgg/controllers/bgg-search.controller.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
   'use strict';
 
-  BGGSearchController.$inject = ['dgBGGService'];
+  BGGSearchController.$inject = ['BggResourceSearch'];
   return BGGSearchController;
 
-  function BGGSearchController(dgBGGService) {
+  function BGGSearchController(bgBggResourceSearch) {
     var vm = this;
+    bgBggResourceSearch.query({search: 'манчкин'}).then(function (BoardGames) {
```
Comment (artyomtrityak) [inline client/static/app/modules/bgg/controllers/bgg-search.controller.js]: why static search?

File: client/static/app/modules/bgg/services/bgg.resource.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    return BGGResourceFactory;
+    function BGGResourceFactory(type) {
+
+      var url = BGG_CONFIG.proxyUrl + '/' + BGG_CONFIG.baseUrl + '/' + BGG_CONFIG.version + '/' + type;
+
+      var transformResponse = function (data) {
+        return x2js.xml_str2json(data);
+      };
+
+      var thenFactoryMethod = function (httpPromise, successcb, errorcb, isArray) {
```
Comment (artyomtrityak) [inline client/static/app/modules/bgg/services/bgg.resource.js]: please make function named without var

**Comment by artyomtrityak** [comment]
merge it

File: client/static/app/modules/bgg/controllers/bgg-search.controller.js (hunk 1 above)
Comment (wsite) [inline client/static/app/modules/bgg/controllers/bgg-search.controller.js]: just for example.

**Pull request closed without merging**